### PR TITLE
Tickets/dm 29502

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -20,10 +20,11 @@ build:
 
 test:
     requires:
-        - ts-conda-build
+        - ts-conda-build =0.3
         - ts-dds
         - ts-idl {{ idl_version }}
         - ts-salobj {{ salobj_version }}
+        - black
```
Comment (couger01) [inline conda/meta.yaml]: Do you need black if it is provided by the metapackage?

**Review by couger01 — CHANGES_REQUESTED** [review]
One question otherwise looks good.

File: conda/meta.yaml (hunk 1 above)
Comment (wvreeven) [reply conda/meta.yaml]: Nope so I removed it.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**